wiki/Website-Changelog.md

```diff
@@ -0,0 +1,172 @@
+# Changelog authoring best practices
+
+- [Tips for writing consistent changelog entries](#tips-for-writing-consistent-changelog-entries)
+- [Content structure](#content-structure)
+  - [Component changes](#component-changes)
+  - [New icons](#new-icons)
+  - [Breaking changes](#breaking-changes)
+- [Templates](#templates)
+  - [Templates for npm packages](#templates-for-npm-packages)
+  - [Templates for Figma changelog entries](#templates-for-figma-changelog-entries)
+
+---
+
+## Tips for writing consistent changelog entries
+
+- Changelog entries should always start with a paragraph (not a list).
+- Consider using a list instead of a long paragraph of text to communicate multiple changes for one element.
+- If you need to include a note (like a migration tip), add it to the bottom of the entry in a new paragraph. Consider using emphasized text, if you need to draw more attention to it.
+- Entries should use complete sentences but be short, clear, and descriptive enough to be helpful.
+- Use past tense.
+- Entries should always end with a period.
+- Use consistent terms for different types of changes: 
+  - Bugfix: “Fixed”
+  - New component, token, variant: “Added”
+  - Update: “Changed”, “Refactored”
+  - Removed or deleted: “Removed”
+  - Other edge cases: “Upgraded”, “Reduced”, “Prevented”
+- When referencing components, use the plain-text component name, e.g., `Accordion`.
+- All components or token names, including those in the summary, should be enclosed in backticks (`).
+
+## Content structure
+
+While best practice guidelines are provided, clarity is most important, so use your best judgment. 
+
+### Component changes
+
+For component changes, place the component name first and enclose it in backticks (`), followed by a dash (-), and then a brief summary.
+
+| Do | Dont |
+|:---|:-----|
+| `Dropdown` - changed the `@height` property to use `max-height` instead of a fixed height.  | Changed the `@height` property in `Dropdown` to use `max-height` instead of a fixed height. |
+
+### New icons
+
+For icons, use a comma-separated list instead of a bulleted list and start with the icon values. Icon names should be enclosed in backticks (`) and use the Ember value.
```

Comment:
I've personal preference for a list, but not a blocker. Just a note.

Reply:
I had that at first, but some of the lists got pretty long, so I changed it. I don't have strong opinions one way or the other, so curious what others think.